### ISSUE TITLE
Error while building in aarch64

### ISSUE THREAD

**Issue by g4uss47** [MythicAgents/merlin#18]
```
./mythic-cli install github https://github.com/MythicAgents/Merlin
2024/05/08 11:29:43 [*] Creating temporary directory
2024/05/08 11:29:43 [*] Cloning https://github.com/MythicAgents/Merlin
Cloning into '/opt/Mythic/tmp'...
2024/05/08 11:29:44 [*] Parsing config.json
[*] Processing Payload Type merlin
[*] merlin already exists. Replace current version?  [y/n]: y
2024/05/08 11:29:45 [*] Stopping current container
2024/05/08 11:29:45 [*] Removing current version
2024/05/08 11:29:45 [+] Successfully removed the current version
2024/05/08 11:29:45 [*] Copying new version of payload into place
2024/05/08 11:29:45 [*] Adding service into docker-compose
WARN[0000] /opt/Mythic/docker-compose.yml: `version` is obsolete
No stopped containers
WARN[0000] /opt/Mythic/docker-compose.yml: `version` is obsolete
[+] Building 2.6s (9/12)                                         docker:default
 => [merlin internal] load build definition from Dockerfile                0.1s
 => => transferring dockerfile: 806B                                       0.0s
 => [merlin internal] load metadata for docker.io/ne0nd0g/merlin-base:v1.  1.0s
 => [merlin internal] load .dockerignore                                   0.0s
 => => transferring context: 2B                                            0.0s
 => [merlin internal] load build context                                   0.1s
 => => transferring context: 447.70kB                                      0.1s
 => [merlin 1/8] FROM docker.io/ne0nd0g/merlin-base:v1.6.1@sha256:d1d4a31  0.0s
 => CACHED [merlin 2/8] WORKDIR /Mythic                                    0.0s
 => CACHED [merlin 3/8] COPY [., .]                                        0.0s
 => CACHED [merlin 4/8] WORKDIR /Mythic/agent                              0.0s
 => ERROR [merlin 5/8] RUN go mod download                                 1.2s
------
 > [merlin 5/8] RUN go mod download:
0.722 exec /bin/sh: exec format error
------
failed to solve: process "/bin/sh -c go mod download" did not complete successfully: exit code: 1
2024/05/08 11:29:50 [+] Successfully installed service
2024/05/08 11:29:50 [*] Skipping over C2 Profile
2024/05/08 11:29:50 [*] Processing Documentation for merlin
[*] merlin documentation already exists. Replace current version?  [y/n]: y
2024/05/08 11:30:00 [*] Removing current version
2024/05/08 11:30:00 [+] Successfully removed the current version
2024/05/08 11:30:00 [*] Copying new documentation into place
2024/05/08 11:30:00 [+] Successfully installed Payload documentation
2024/05/08 11:30:00 [*] Skipping over C2 Documentation
2024/05/08 11:30:00 [*] Skipping over Wrapper Documentation
```

**Comment by Ne0nd0g** [comment]
@g4uss47 are you running this on a macOS host with an M-series CPU?

**Comment by g4uss47** [comment]
It is being run on a Raspberry Pi with Raspbian 64 bit OS

**Comment by Ne0nd0g** [comment]
OK, I'll look into it. At first glance this looks more like a problem with Docker and not Merlin.

**Comment by huobb0** [comment]
Any update on this issue? Trying to get around this problem on an M-series MAC.

**Comment by Ne0nd0g** [comment]
@huobb0 I believe I know what the fix is. I'm going to get it pushed out before the end of the day.

**Comment by Ne0nd0g** [comment]
Almost there; I have a verified fix but need to push changes to other repos first.

**Comment by Ne0nd0g** [comment]
@huobb0 for a temporary work around, change the first line of the Dockerfile located at `Mythic/InstalledServices/merlin/Dockerfile` to `FROM ne0nd0g/merlin-base:v1.7.0`

**Comment by Ne0nd0g** [comment]
Fixed in 2dcb178b2ad64e9703a97facd2a32c6419ee06dc
